Generate.py

```diff
@@ -440,7 +441,11 @@ def roll_settings(weights: dict, plando_options: PlandoOptions = PlandoOptions.b
 
     ret.game = get_choice("game", weights)
     if ret.game not in AutoWorldRegister.world_types:
-        picks = Utils.get_fuzzy_results(ret.game, AutoWorldRegister.world_types, limit=1)[0]
+        picks = Utils.get_fuzzy_results(ret.game, list(AutoWorldRegister.world_types) + failed_world_loads, limit=1)[0]
+        if picks[0] in failed_world_loads:
+            raise Exception(f"No functional world found to handle game {ret.game}. "
+                            f"Did you mean '{picks[0]}' ({picks[1]}% sure)? "
+                            f"If so, it appears the world failed to initialize correctly.")
         raise Exception(f"No world found to handle game {ret.game}. Did you mean '{picks[0]}' ({picks[1]}% sure)? "
                         f"Check your spelling or installation of that world.")
```

Comment:
Feels like I'm nitpicking, but is it still worth mentioning the installation here, since the failed worlds are handled above?

Reply:
Idr exactly, but wasn't fuzzy matching case sensitive? If so, Factorio <-> factorio would be unable to match the 'F', which makes it less likely to find a good result.
Another idea to make this better would be to print "X worlds failed to load" in both code paths if  failed_world_loads isn't empty.